modules/legal/pages/copyright-uyuni.adoc

```diff
@@ -0,0 +1,7 @@
+= Copyright Notice
```

Comment:
```suggestion
= Copyright Notice
ifeval::[{mlm-content} == true]
:noindex:
endif::[]
```

Reply:
I thinnk we'd better add this `ifeval`, and change the mlm copyright file accordingly.  Please, review #3910.